src/ec/packages/ec-component-breadcrumb-standardised/__snapshots__/breadcrumb-standardised.test.js.snap

```diff
@@ -0,0 +1,602 @@
+      </a>
+      <svg
+        aria-hidden="true"
+        class="ecl-icon ecl-icon--2xs ecl-icon--rotate-90 ecl-breadcrumb-standardised__icon"
+        focusable="false"
+        role="presentation"
+      >
+        <use
+          xlink:href="static/icons.svg#ui--corner-arrow"
+        />
+      </svg>
+    </li>
+    <li
+      aria-hidden="false"
+      class="ecl-breadcrumb-standardised__segment"
+      data-ecl-breadcrumb-standardised-item="static"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-breadcrumb-standardised__link"
+        href="/example"
+      >
+        How the Commission is organised
+      </a>
+      <svg
+        aria-hidden="true"
+        class="ecl-icon ecl-icon--2xs ecl-icon--rotate-90 ecl-breadcrumb-standardised__icon"
+        focusable="false"
+        role="presentation"
+      >
+        <use
+          xlink:href="static/icons.svg#ui--corner-arrow"
+        />
+      </svg>
+    </li>
+    <li
+      aria-current="page"
+      aria-hidden="false"
+      class="ecl-breadcrumb-standardised__segment ecl-breadcrumb-standardised__current-page"
+      data-ecl-breadcrumb-standardised-item="static"
+    >
+      News
+    </li>
+  </ol>
+</nav>
+`;
+
+exports[`EC - Breadcrumb Standardised Long renders correctly with extra attributes 1`] = `
+<nav
+  aria-label="You are here:"
+  class="ecl-breadcrumb-standardised"
+  data-ecl-auto-init="BreadcrumbStandardised"
+  data-ecl-breadcrumb-standardised="true"
+  data-test="data-test-value"
+  data-test-1="data-test-value-1"
+>
+  <ol
+    class="ecl-breadcrumb-standardised__container"
+  >
+    <li
+      aria-hidden="false"
+      class="ecl-breadcrumb-standardised__segment"
+      data-ecl-breadcrumb-standardised-item="static"
+    >
+      <a
+        class="ecl-link ecl-link--standalone ecl-breadcrumb-standardised__link"
+        href="/example"
+      >
+        Home
+      </a>
+      <svg
+        aria-hidden="true"
+        class="ecl-icon ecl-icon--2xs ecl-icon--rotate-90 ecl-breadcrumb-standardised__icon"
+        focusable="false"
+        role="presentation"
+      >
+        <use
+          xlink:href="static/icons.svg#ui--corner-arrow"
+        />
+      </svg>
+    </li>
+    <li
+      aria-hidden="false"
+      class="ecl-breadcrumb-standardised__segment ecl-breadcrumb-standardised__segment--ellipsis"
+      data-ecl-breadcrumb-standardised-ellipsis="true"
+    >
+      <button
+        aria-label="Click to expand"
+        class="ecl-breadcrumb-standardised__ellipsis"
+        data-ecl-breadcrumb-standardised-ellipsis-button="true"
+        type="button"
+      >
+        …
+      </button>
+      <svg
+        aria-hidden="true"
+        class="ecl-icon ecl-icon--2xs ecl-icon--rotate-90 ecl-breadcrumb-standardised__icon"
+        focusable="false"
+        role="presentation"
+      >
+        <use
+          xlink:href="static/icons.svg#ui--corner-arrow"
+        />
+      </svg>
+    </li>
+    <li
+      aria-hidden="true"
```

Comment:
The aria-attributes are set by the javascript when needed, they are used to hide and show content based on the script, meaning the size of the view port, the length of the breadcrumb, if javascript acts, these attributes are set otherwise they are not and everything is going to be visible.